src/app/messages/messages.component.ts

```diff
@@ -0,0 +1,16 @@
+import { Component, OnInit } from '@angular/core';
+import { MessageService } from '../message.service';
+
+@Component({
+  selector: 'app-messages',
+  templateUrl: './messages.component.html',
+  styleUrls: ['./messages.component.css']
+})
+export class MessagesComponent implements OnInit {
+
+  constructor(public messageService: MessageService) { }
+
+  ngOnInit() {
+  }
```

Comment:
Remove lifecycle implementation if not required.

Reply:
export class MessagesComponent {

  constructor(public messageService: MessageService) { }

}